code/modules/reagents/chemistry/reagents/drinks_reagents.dm

```diff
@@ -183,8 +183,15 @@
 	if(HAS_TRAIT(M, TRAIT_COMIC_SANS) || issmall(M))
 		update_flags |= M.adjustBruteLoss(-1, FALSE)
 		update_flags |= M.adjustFireLoss(-1, FALSE)
+	if(M.mind.assigned_role == "Clown")
```

Comment:
Codewise is fine, but I don't like the idea of this being a cling-check

Reply:
changed so its not